tox.ini

```diff
@@ -0,0 +1,19 @@
+[tox]
+requires =
+    tox>=4.2
+env_list =
+    py{312, 311, 310, 39, 38}
+
+[testenv]
+pass_env =
+    FORCE_COLOR
+commands_pre =
+    {envpython} -m pip install -U -r dev-requirements.txt
+commands =
+    {envpython} -m pytest \
+      --cov blurb_it \
+      --cov tests \
+      --cov-report html \
+      --cov-report term \
+      --cov-report xml \
```

Comment:
Could we add these options to our `.coveragerc` file instead, so people get the same config enabled even if they don't use tox?

Reply:
I'm not sure about the `.coveragerc` options, so moved to `pytest.ini`, like in https://github.com/python/peps/blob/main/pytest.ini

Reply:
if it works, it works!